packages/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -277,4 +283,80 @@ describe("TinyliciousClient", () => {
         await corruptedAliasOp(runtimeOf(do1), "alias");
         assert(await dataCorruption);
     });
+
+    /**
+     * Scenario: Test if TinyliciousClient with only read permission starts the container in read mode.
+     * TinyliciousClient will attempt to start the connection in write mode, but if the write permission
+     * is not available or the file is read only, the connection mode will be read.
+     *
+     * Expected behavior: TinyliciousClient should start the container with the connectionMode in `read`
+     */
+    it("can create a container with only read permission in read mode", async () => {
+        const tokenProvider = new InsecureTinyliciousTokenProvider([ScopeType.DocRead]);
+        const client = new TinyliciousClient({ connection: { tokenProvider } });
+
+        const { container } = await client.createContainer(schema);
+        const containerId = await container.attach();
+        await timeoutPromise(
+            (resolve) => container.once("connected", resolve),
+            {
+                durationMs: 1000,
+                errorMsg: "container connect() timeout",
+            },
+        );
+        const { container: containerGet } = await client.getContainer(
+            containerId,
+            schema,
+        );
+
+        assert.strictEqual(
+            connectionModeOf(container),
+            "read",
+            "Creating a container with only read permission is not in read mode",
+        );
+
+        assert.strictEqual(
+            connectionModeOf(containerGet),
+            "read",
+            "Getting a container with only read permission is not in read mode",
+        );
+    });
+
+    /**
+     * Scenario: Test if TinyliciousClient with read and write permissions starts the container in write mode.
+     * TinyliciousClient will attempt to start the connection in write mode, but if the file is read only,
+     * the connection mode will be read.
```

Comment:
Can you adjust this to reflect something like this: "TinyliciousClient will attempt to start the connection in write mode, and since access permissions offer write capability, the established connection mode will be "write".